index.html

```diff
@@ -96,11 +96,11 @@ <h1>
         </div>
       </div>
     </section>
-    <section class="projects" id="my-projects">
+    <div class="projects">
       <h2>My Recent Works</h2>
       <span class="line"></span>
-      <div class="project-cards">
-        <div class="project-card">
+       <div class="project-cards" id="my-portfolio">
+        <!-- <div class="project-card">
```

Comment:
- [ ] As mentioned in the project requirements, kindly make sure your project section is rendered dynamically through javascript. so make sure you delete it here in your markup and render it with javascript. As i can see you have commented this section, please make sure you don;t have the commented lines of code in your code base. you may consider dealing them